app/services/reinstate_conditions_met.rb

```diff
@@ -25,7 +25,7 @@ def save
         ActiveRecord::Base.transaction do
           ApplicationStateChange.new(application_choice).reinstate_conditions_met!
           application_choice.update(attrs)
-          UpdateOfferConditions.new(application_choice: application_choice).call
+          application_choice.offer.conditions.update_all(status: :met)
```

Comment:
Can we use `update` rather than `update_all` here. `update_all` skips callbacks (and hence auditing which is what we are making this change for!).

Reply:
Ah I missed this as it mentioned updating in bulk in the title. Sorry @richardpattinson!

Reply:
hahahah thats fair, ill make the change now